api/external/applications/applications.proto

```diff
@@ -398,7 +398,7 @@ message Service {
   // Intentionally blank.
   HealthStatus health_check = 4;
   // Intentionally blank.
```

Comment:
```suggestion
```

This comment isn't necessary anymore.

Reply:
When we originally annotated all the things in this file we put `// Unused.` for the other `reserved` thing so I did that instead for consistency. 👍